graphql/resolvers.test.js

```diff
@@ -129,3 +129,57 @@ describe('Alerts resolvers', () => {
     ])
   })
 })
+
+describe('userInfo resolver', () => {
+  test('should return null if no username is passed in', async () => {
```

Comment:
We may want this to throw an error, I believe that is the behavior of other resolvers when the args does not match with what is expected